examples/3d/meshlet.rs

```diff
@@ -0,0 +1,114 @@
+//! GPU-driven meshlet-based rendering.
+
+#[path = "../helpers/camera_controller.rs"]
+mod camera_controller;
+
+use bevy::{
+    pbr::{
+        meshlet::{MaterialMeshletMeshBundle, MeshletPlugin},
+        CascadeShadowConfigBuilder,
+    },
+    prelude::*,
+    render::render_resource::AsBindGroup,
+};
+use camera_controller::{CameraController, CameraControllerPlugin};
+use std::f32::consts::PI;
+
+fn main() {
+    App::new()
+        .add_plugins((
+            DefaultPlugins,
+            MeshletPlugin,
+            MaterialPlugin::<MeshletDebugMaterial>::default(),
+            CameraControllerPlugin,
+        ))
+        .add_systems(Startup, setup)
+        .run();
+}
+
+fn setup(
+    mut commands: Commands,
+    asset_server: Res<AssetServer>,
+    mut standard_materials: ResMut<Assets<StandardMaterial>>,
+    mut debug_materials: ResMut<Assets<MeshletDebugMaterial>>,
+) {
+    commands.spawn((
+        Camera3dBundle {
+            transform: Transform::from_translation(Vec3::new(1.8194779, 0.47882894, -0.08456142))
+                .with_rotation(Quat::from_array([
+                    -0.10061985,
+                    0.7282399,
+                    0.10953168,
+                    0.6689881,
+                ])),
```

Comment:
```suggestion
            transform: Transform::from_translation(Vec3::new(1.8, 0.4, -0.1))
                .looking_at(Vec3::ZERO, Vec3::Y),
```
that transform looks less scary to me 🙂